process/project-lifecycle.md

```diff
@@ -0,0 +1,159 @@
+* Nurture Projects (e.g. via OpenSSF Service Desk requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct and IP Policy requirements.
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Sandbox Entry Requirements
+
+Projects being submitted to the OpenSSF at the sandbox level are intended to be the entry point for early stage projects and are not required to undergo due diligence.
+
+Sandbox Projects should be early-stage projects that the OpenSSF TAC or parent Working Group believes warrant experimentation.
+
+* New Projects that are designed to extend one or more OpenSSF Projects with functionality or interoperability libraries.
+* Independent projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+* Projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
+* Any project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that
+
+#### Project Graduation Process: Sandbox to Incubating
+
+Incubating Projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor, with two weeks for public comment before a vote is called.
+
+#### Submission Forms for Sandbox
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+#### Submission Forms for Sandbox to Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+### Incubating
+
+![Incubating](/process/incubation-process.png)
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Incubating Entry Requirements
+
+To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the TAC’s judgement, are of adequate quality and scope.
+* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgement over the level of activity that is adequate to meet these criteria
+* A clear versioning scheme.
+* Specifications must have at least one public reference implementation.
+
+#### Project Graduation Process: Incubating to Graduation
+
+Projects that wish to move from Incubating to Graduation should open a PR confirming the following criteria:
+* Have committers from at least two organizations.
```

Comment:
```suggestion
* Have committers from at least three independent (where one is not a subsidiary of the other) organizations.
```

Reply:
Yes, I'm proposing the bar be set at 3, which is where the CNCF originally set the bar. Happy to discuss this as it is a significant change.

Reply:
3 feels too strong. Maybe 2 with a few other open source contributors. Happy to discuss this in tac meeting, but seems fine to leave 2 for now.